spec.html

```diff
@@ -41607,12 +41607,11 @@ <h1>Lexical Grammar</h1>
     <h1>Expressions</h1>
     <emu-prodref name=IdentifierReference></emu-prodref>
     <emu-prodref name=BindingIdentifier></emu-prodref>
-    <emu-prodref name=Identifier></emu-prodref>
-    <emu-prodref name=AsyncArrowBindingIdentifier></emu-prodref>
     <emu-prodref name=LabelIdentifier></emu-prodref>
+    <emu-prodref name=Identifier></emu-prodref>
     <emu-prodref name=PrimaryExpression></emu-prodref>
     <emu-prodref name=CoverParenthesizedExpressionAndArrowParameterList></emu-prodref>
-    <p>When processing an instance of the production <emu-prodref name=PrimaryExpression a="parencover"></emu-prodref> the interpretation of |CoverParenthesizedExpressionAndArrowParameterList| is refined using the following grammar:</p>
```

Comment:
i'd prefer we quote all attribute values rather than removing quotes

Reply:
That seems like it ought to be a separate PR. In this PR it makes sense to match the prevailing editorial style.

Reply:
Even only in terms of section IDs, I count 2153 `="sec` versus 3 `=sec`; when I search for `=[^"]` versus `="` i get 668 (which includes a bunch of equals signs throughout the text) versus 4243.

The prevailing editorial style seems to be, use quotes.

Reply:
@bakkot probably meant that quoteless is (by far) the prevailing style *within Annex A*. But I'm fine to change it to quoted.

Reply:
If the intention is to reduce churn, let's either leave them as is, or bring them closer to the common style, rather than making this annex less consistent with the larger spec?

Reply:
Leaving them as-is would defeat (or at least go against) the purpose of this PR. Changing all of Annex A to quoted would certainly touch more lines (~300 vs 14), but I don't think it would be much of a problem: it seems like very few PRs touch Annex A (even when they should), so I suspect very few in-flight PRs would get merge conflicts. Like I say, I'm okay with adding quotes in Annex A.

Reply:
Flipped the second commit to add quotes to all unquoted attribute values.